source/deployment/ldap-group-sync.rst

```diff
@@ -137,7 +137,7 @@ Users who have logged in and accessed Mattermost will be visible in the members
 
 .. image:: ../images/Group_Members.png
 
-Users can be removed from the Mattermost group on subsequent synchronizations. However, they will not be removed from teams and channels.
+Users can be removed from the Mattermost group on subsequent synchronizations.
```

Comment:
Alternatively we could say "However, they will not be removed from teams and channels unless the team or channel is group-synced.

Reply:
@thefactremains I like Martin's edit as its a bit more explicit.